integration/package.json

```diff
@@ -1,14 +1,21 @@
 {
-  "name": "integration-tests",
+  "name": "integration",
   "version": "0.0.0",
   "private": true,
   "description": "deps needed for integration tests",
   "type": "module",
-  "scripts": {},
+  "scripts": {
+    "tsc": "tsc"
```

Comment:
Note to self: Can this go back to being called `typecheck` now that templates aren't in the workspace?